src/Microsoft.Azure.SignalR.AspNet/Infrastructure/ServiceConnection.cs

```diff
@@ -69,54 +72,56 @@ protected override Task CleanupConnections()
 
         protected override Task OnConnectedAsync(OpenConnectionMessage openConnectionMessage)
         {
+            // Create channel per client to async process message
+            var connectionId = openConnectionMessage.ConnectionId;
+            var channel = Channel.CreateUnbounded<ServiceMessage>();
+            channel.Writer.TryWrite(openConnectionMessage);
```

Comment:
>TryWrite [](start = 27, length = 8)

other places is `await WriteAsync` while here uses `TryWrite`, any concerns?

Reply:
TryWrite() means attempt to read or write an item synchronously. To make sure open connection message is first priority before rest. 

---
In reply to: [237729148](https://github.com/Azure/azure-signalr/pull/268#discussion_r237729148) [](ancestors = 237729148)